### ISSUE TITLE
combined_metadata has some munged colnames

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/curatedMetagenomicData#159]
Present in both release and devel. Only the underscore versions are present in the [template](https://github.com/waldronlab/curatedMetagenomicDataCuration/blob/master/inst/extdata/template.csv) so this is not a curation error.

``` r

suppressPackageStartupMessages(library(dplyr))
suppressPackageStartupMessages(library(curatedMetagenomicData))
problemcols <- !isUnique(sub(".", "_", colnames(combined_metadata), fixed=TRUE))
problemcolnames <- sort(colnames(combined_metadata)[problemcols])
select(combined_metadata, one_of(problemcolnames)) %>%
  mutate_if(sapply(., is.character), as.factor) %>%
  summary()
#>    c_peptide       c.peptide         fgf_19           fgf.19     
#>  Min.   :0.040   Min.   : NA     Min.   : 16.40   Min.   : NA    
#>  1st Qu.:1.600   1st Qu.: NA     1st Qu.: 71.64   1st Qu.: NA    
#>  Median :2.132   Median : NA     Median :107.83   Median : NA    
#>  Mean   :2.358   Mean   :NaN     Mean   :138.50   Mean   :NaN    
#>  3rd Qu.:2.913   3rd Qu.: NA     3rd Qu.:160.03   3rd Qu.: NA    
#>  Max.   :8.408   Max.   : NA     Max.   :782.18   Max.   : NA    
#>  NA's   :9974    NA's   :10199   NA's   :10069    NA's   :10199  
#>           flg_genotype   flg.genotype     glp_1            glp.1      
#>  e2422x         :    2   NA's:10199   Min.   : 0.000   Min.   : NA    
#>  e2422x/3321dela:    2                1st Qu.: 0.300   1st Qu.: NA    
#>  s1515x         :    2                Median : 0.900   Median : NA    
#>  wt             :   72                Mean   : 1.774   Mean   :NaN    
#>  NA's           :10121                3rd Qu.: 1.700   3rd Qu.: NA    
#>                                       Max.   :17.800   Max.   : NA    
#>                                       NA's   :10069    NA's   :10199  
#>       il_1             il.1      
#>  Min.   : 0.000   Min.   : NA    
#>  1st Qu.: 0.000   1st Qu.: NA    
#>  Median : 0.000   Median : NA    
#>  Mean   : 0.982   Mean   :NaN    
#>  3rd Qu.: 0.000   3rd Qu.: NA    
#>  Max.   :16.850   Max.   : NA    
#>  NA's   :10069    NA's   :10199
```

<sup>Created on 2019-12-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

<details>

<summary>Session info</summary>

``` r
devtools::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value                       
#>  version  R version 3.6.1 (2019-07-05)
#>  os       Ubuntu 18.04.3 LTS          
#>  system   x86_64, linux-gnu           
#>  ui       X11                         
#>  language (EN)                        
#>  collate  en_US.UTF-8                 
#>  ctype    en_US.UTF-8                 
#>  tz       America/New_York            
#>  date     2019-12-06                  
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package                * version date       lib source        
#>  AnnotationDbi            1.48.0  2019-10-29 [1] Bioconductor  
#>  AnnotationHub          * 2.18.0  2019-10-29 [1] Bioconductor  
#>  assertthat               0.2.1   2019-03-21 [1] CRAN (R 3.6.1)
#>  backports                1.1.5   2019-10-02 [1] CRAN (R 3.6.1)
#>  Biobase                * 2.46.0  2019-10-29 [1] Bioconductor  
#>  BiocFileCache          * 1.10.2  2019-11-08 [1] Bioconductor  
#>  BiocGenerics           * 0.32.0  2019-10-29 [1] Bioconductor  
#>  BiocManager              1.30.10 2019-11-16 [1] CRAN (R 3.6.1)
#>  BiocVersion              3.10.1  2019-06-06 [1] Bioconductor  
#>  bit                      1.1-14  2018-05-29 [1] CRAN (R 3.6.1)
#>  bit64                    0.9-7   2017-05-08 [1] CRAN (R 3.6.1)
#>  blob                     1.2.0   2019-07-09 [1] CRAN (R 3.6.1)
#>  callr                    3.3.2   2019-09-22 [1] CRAN (R 3.6.1)
#>  cli                      1.1.0   2019-03-19 [1] CRAN (R 3.6.1)
#>  crayon                   1.3.4   2017-09-16 [1] CRAN (R 3.6.1)
#>  curatedMetagenomicData * 1.16.0  2019-11-05 [1] Bioconductor  
#>  curl                     4.3     2019-12-02 [1] CRAN (R 3.6.1)
#>  DBI                      1.0.0   2018-05-02 [1] CRAN (R 3.6.1)
#>  dbplyr                 * 1.4.2   2019-06-17 [1] CRAN (R 3.6.1)
#>  desc                     1.2.0   2018-05-01 [1] CRAN (R 3.6.1)
#>  devtools                 2.2.1   2019-09-24 [1] CRAN (R 3.6.1)
#>  digest                   0.6.23  2019-11-23 [1] CRAN (R 3.6.1)
#>  dplyr                  * 0.8.3   2019-07-04 [1] CRAN (R 3.6.1)
#>  ellipsis                 0.3.0   2019-09-20 [1] CRAN (R 3.6.1)
#>  evaluate                 0.14    2019-05-28 [1] CRAN (R 3.6.1)
#>  ExperimentHub          * 1.12.0  2019-10-29 [1] Bioconductor  
#>  fastmap                  1.0.1   2019-10-08 [1] CRAN (R 3.6.1)
#>  fs                       1.3.1   2019-05-06 [1] CRAN (R 3.6.1)
#>  glue                     1.3.1   2019-03-12 [1] CRAN (R 3.6.1)
#>  highr                    0.8     2019-03-20 [1] CRAN (R 3.6.1)
#>  htmltools                0.4.0   2019-10-04 [1] CRAN (R 3.6.1)
#>  httpuv                   1.5.2   2019-09-11 [1] CRAN (R 3.6.1)
#>  httr                     1.4.1   2019-08-05 [1] CRAN (R 3.6.1)
#>  interactiveDisplayBase   1.24.0  2019-10-29 [1] Bioconductor  
#>  IRanges                  2.20.1  2019-11-20 [1] Bioconductor  
#>  knitr                    1.26    2019-11-12 [1] CRAN (R 3.6.1)
#>  later                    1.0.0   2019-10-04 [1] CRAN (R 3.6.1)
#>  lifecycle                0.1.0   2019-08-01 [1] CRAN (R 3.6.1)
#>  magrittr                 1.5     2014-11-22 [1] CRAN (R 3.6.1)
#>  memoise                  1.1.0   2017-04-21 [1] CRAN (R 3.6.1)
#>  mime                     0.7     2019-06-11 [1] CRAN (R 3.6.1)
#>  pillar                   1.4.2   2019-06-29 [1] CRAN (R 3.6.1)
#>  pkgbuild                 1.0.6   2019-10-09 [1] CRAN (R 3.6.1)
#>  pkgconfig                2.0.3   2019-09-22 [1] CRAN (R 3.6.1)
#>  pkgload                  1.0.2   2018-10-29 [1] CRAN (R 3.6.1)
#>  prettyunits              1.0.2   2015-07-13 [1] CRAN (R 3.6.1)
#>  processx                 3.4.1   2019-07-18 [1] CRAN (R 3.6.1)
#>  promises                 1.1.0   2019-10-04 [1] CRAN (R 3.6.1)
#>  ps                       1.3.0   2018-12-21 [1] CRAN (R 3.6.1)
#>  purrr                    0.3.3   2019-10-18 [1] CRAN (R 3.6.1)
#>  R6                       2.4.1   2019-11-12 [1] CRAN (R 3.6.1)
#>  rappdirs                 0.3.1   2016-03-28 [1] CRAN (R 3.6.1)
#>  Rcpp                     1.0.3   2019-11-08 [1] CRAN (R 3.6.1)
#>  remotes                  2.1.0   2019-06-24 [1] CRAN (R 3.6.1)
#>  rlang                    0.4.2   2019-11-23 [1] CRAN (R 3.6.1)
#>  rmarkdown                1.18    2019-11-27 [1] CRAN (R 3.6.1)
#>  rprojroot                1.3-2   2018-01-03 [1] CRAN (R 3.6.1)
#>  RSQLite                  2.1.3   2019-12-03 [1] CRAN (R 3.6.1)
#>  S4Vectors                0.24.1  2019-12-01 [1] Bioconductor  
#>  sessioninfo              1.1.1   2018-11-05 [1] CRAN (R 3.6.1)
#>  shiny                    1.4.0   2019-10-10 [1] CRAN (R 3.6.1)
#>  stringi                  1.4.3   2019-03-12 [1] CRAN (R 3.6.1)
#>  stringr                  1.4.0   2019-02-10 [1] CRAN (R 3.6.1)
#>  testthat                 2.3.1   2019-12-01 [1] CRAN (R 3.6.1)
#>  tibble                   2.1.3   2019-06-06 [1] CRAN (R 3.6.1)
#>  tidyr                    1.0.0   2019-09-11 [1] CRAN (R 3.6.1)
#>  tidyselect               0.2.5   2018-10-11 [1] CRAN (R 3.6.1)
#>  usethis                  1.5.1   2019-07-04 [1] CRAN (R 3.6.1)
#>  vctrs                    0.2.0   2019-07-05 [1] CRAN (R 3.6.1)
#>  withr                    2.1.2   2018-03-15 [1] CRAN (R 3.6.1)
#>  xfun                     0.11    2019-11-12 [1] CRAN (R 3.6.1)
#>  xtable                   1.8-4   2019-04-21 [1] CRAN (R 3.6.1)
#>  yaml                     2.2.0   2018-07-25 [1] CRAN (R 3.6.1)
#>  zeallot                  0.1.0   2018-01-28 [1] CRAN (R 3.6.1)
#> 
#> [1] /usr/local/lib/R/site-library
#> [2] /usr/lib/R/site-library
#> [3] /usr/lib/R/library
```

</details>

**Comment by lwaldron** [comment]
Confirmed this bug is still active.

**Comment by schifferl** [comment]
Closed by 7489877 (in devel) and 0e7bbc8 (in release), both have been pushed to bioc. A `family_member` column should be added to the template.
